app/controllers/home/oauths_controller.rb

```diff
@@ -8,17 +9,20 @@ def oauth
   def callback
     provider = auth_params[:provider]
     if @user = login_from(provider)
-      redirect_to trainer_path, notice: (t 'log_in_is_successful_provider_notice',
+      redirect_to trainer_path,
+                  notice: (t 'log_in_is_successful_provider_notice',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.